orttraining/orttraining/python/orttraining_pybind_state.cc

```diff
@@ -162,6 +162,35 @@ struct TrainingConfigurationResult {
   optional<std::string> loss_scale_input_name;
 };
 
+#ifdef ENABLE_TRAINING_ON_DEVICE
```

Comment:
Please refer to PR #13964 which might change the use of this macro. Whichever PR goes in later might need to update.

Reply:
do we need to change this after the other PR gets merged or before,
so it should just be in ENABLE_TRAINING flag from what I understand from the PR.

Reply:
No, it should be ENABLE_TRAINING_APIS...